### PR TITLE
ReferencedResourceManager compatibility update

### PR DESCRIPTION
**Pull request by nathanholzworth**
Updated the index.js and supporting files to use the new ReferencedResourceManager with MustacheTemplate Controller.

### CODE REVIEW

**Comment by nathanholzworth** [comment]
This PR depends on https://github.com/razee-io/razeedeploy-core/pull/347
Please wait till that PR is merged before merging this one. src/index.js will also need a small update to its requires to point it to the ReferencedResourceManager

**Review by alewitt2 — APPROVED** [review]

**Pull request merged**
